### PR TITLE
Updated websocket client invalid view

### PR DESCRIPTION
**Pull request by ashamis**

### CODE REVIEW

File: python/ccf/clients.py [hunk 1]
```diff
@@ -174,6 +174,13 @@ def get_curve(ca_file):
     )
 
 
+def unpack_seqno_or_view(data):
+    (value,) = struct.unpack("<q", data)
+    if value == -1:
```
Comment (jumaffre) [inline python/ccf/clients.py:179]: `seqno` default is `kv::NoVersion` (`std::numeric_limits<Version>::min()`) while `view` default is `-1`. I think each of those should have a different path for these edge-case values.

Comment (achamayou) [reply python/ccf/clients.py:179]: Why not make both -1?

File: python/ccf/clients.py [hunk 2]
```diff
@@ -174,6 +174,13 @@ def get_curve(ca_file):
     )
 
 
+def unpack_seqno_or_view(data):
+    (value,) = struct.unpack("<q", data)
```
Comment (achamayou) [inline python/ccf/clients.py:178]:
```suggestion
    value, = struct.unpack("<q", data)
```

File: tests/infra/checker.py [hunk 3]
```diff
@@ -34,7 +34,7 @@ def __call__(self, rpc_result, result=None, error=None, timeout=2):
                     result, rpc_result.body
                 )
 
-            assert rpc_result.seqno and rpc_result.view, rpc_result
+            assert rpc_result.seqno and rpc_result.view >= 0, rpc_result
```
Comment (achamayou) [inline tests/infra/checker.py:37]:
```suggestion
            assert rpc_result.seqno >= 0 and rpc_result.view >= 0, rpc_result
```

File: python/ccf/clients.py (hunk 2 above)
Comment (ashamis) [reply python/ccf/clients.py:178]: The braces are enforced by black

File: python/ccf/clients.py (hunk 1 above)
Comment (ashamis) [reply python/ccf/clients.py:179]: I changed kv::NoVersion to be -1

File: python/ccf/clients.py (hunk 2 above)
Comment (achamayou) [reply python/ccf/clients.py:178]: Ah, yes!

**Review by achamayou — APPROVED** [review]

File: python/ccf/clients.py (hunk 1 above)
Comment (achamayou) [reply python/ccf/clients.py:179]: As pointed by @jumaffre separately, we can't do that because we do use negative versions internally, so we should use `std::numeric_limits<Version>::min()` for both.

File: src/node/rpc/test/tx_status_test.cpp [hunk 4]
```diff
@@ -65,12 +65,18 @@ TEST_CASE("edge cases")
...
     // Impossible: view for all global txs must be known
     // get_tx_status(a, N, 0, b, >=N)
-    CHECK_THROWS(get_tx_status(3, 10, -1, 1, 10));
-    CHECK_THROWS(get_tx_status(3, 10, -1, 1, 11));
-    CHECK_THROWS(get_tx_status(3, 10, -1, 3, 10));
-    CHECK_THROWS(get_tx_status(3, 10, -1, 3, 11));
-    CHECK_THROWS(get_tx_status(3, 10, -1, 4, 10));
-    CHECK_THROWS(get_tx_status(3, 10, -1, 4, 11));
+    CHECK_THROWS(
+      get_tx_status(3, 10, std::numeric_limits<int64_t>::min(), 1, 10));
```
Comment (jumaffre) [inline src/node/rpc/test/tx_status_test.cpp:69]: Minor: Use `VIEW_UNKNOWN` instead of `std::numeric_limits<int64_t>::min()`?
